paper/paper.md

```diff
@@ -0,0 +1,70 @@
+tags:
+  - C++
+  - Python
+  - astronomy
+  - dynamics
+  - asteroids
+  - gravity
+  - numerical methods
+  - polyhedral model
+authors:
+  - name: Jonas Schuhmacher
+    orcid: 0009-0005-9693-4530
+    affiliation: 1
+  - name: Emmanuel Blazquez
+    orcid: 0000-0001-9697-582X
+    affiliation: 2
+  - name: Fabio Gratl
+    orcid: 0000-0001-5195-7919
+    affiliation: 1
+  - name: Dario Izzo
+    orcid: 0000-0002-9846-8423
+    affiliation: 2
+  - name: Pablo Gómez
+    orcid: 0000-0002-5631-8240
+    affiliation: 2
+affiliations:
+ - name: Technische Universität München, Arcisstraße 21, 80333 München, Germany 
+   index: 1
+ - name: Advanced Concepts Team, European Space Agency, European Space Research and Technology Centre (ESTEC), Keplerlaan 1, 2201 AZ Noordwijk, The Netherlands
+   index: 2
+date: 07 July 2023
+bibliography: paper.bib
+
+---
+
+# Summary
+
+Polyhedral gravity models are ubiquitous for modeling the gravitational field of irregular bodies, such as asteroids and comets. We present an open-source C++ library for the efficient, parallelized computation of a polyhedral gravity model. We also provide a Python interface to the library using *pybind11*. The library is particularly focused on delivering high performance and scalability which we achieve through vectorization and parallelization with *xsimd* and *thrust*, respectively. The library supports many common formats, such as *.stl*, *.off*, *.ply*, *.mesh* and *tetgen*'s *.node* and *.face*.
+
+# Statement of Need
+
+The complex gravitational fields of irregular bodies, such as asteroids and comets, are often modeled using polyhedral gravity models as they provide an analytic solution for the computation of the gravitational potential, acceleration (and second derivative) given a mesh of the body [@tsoulis2012analytical;@tsoulis2021computational]. The computation of the gravitational potential and acceleration is a computationally expensive task, especially for large meshes, which can however benefit from parallelization either over computed targets points for which we seek potential and acceleration or over the mesh. Thus, a high-performance implementation of a polyhedral gravity model is desirable. 
+
+While some research code for these models exists, they are not focused on usability and limited to FORTRAN **TODO LINK** and proprietary software like MATLAB **TODO LINK**. There is a lack of well-documented, maintained open-source implementations, particularly in modern programming languages and with a focus on scalability and performance.
+
+The presented software has already seen application in several research works. It has been used to optimize trajectories around the highly irregular comet 67P/Churyumov-Gerasimenko [@marak2023trajectory]. Further, it has been used to study the effectiveness of so-called neural density fields [@izzo2022geodesy], where it can serve as a ground truth and to pretrain neural networks [@GeodesyNetsBenchmark]. **TODO_add_more_examples**
```

Comment:
```suggestion
The presented software has already seen application in several research works. It has been used to optimize trajectories around the highly irregular comet 67P/Churyumov-Gerasimenko [@marak2023trajectory]. Further, it has been used to study the effectiveness of so-called neural density fields [@izzo2022geodesy], where it can serve as ground truth and to pre-train neural networks [@GeodesyNetsBenchmark]. **TODO_add_more_examples**
```